### PR TITLE
fix: replaced \n with line break in Schema

### LINKED ISSUE #1264: [TASK] Documentation Combine Settings \n plain text instead of new line
### Summary:

Schema `Combine` attribute types (https://dynamoosejs.com/guide/Schema#attribute-types) `Settings` has `\n` text instead of an actual new line.

### Other:
- [x] I have read through the Dynamoose documentation before posting this issue
- [x] I have searched through the GitHub issues (including closed issues) and pull requests to ensure this issue has not already been raised before
- [x] I have searched the internet and Stack Overflow to ensure this issue hasn't been raised or answered before
- [x] I have tested the code provided and am confident it doesn't work as intended
- [x] I have filled out all fields above
- [x] I am running the latest version of Dynamoose

### PR DESCRIPTION
**Pull request by MukulKolpe**
### Summary:

replaced /n with line break in the Combine section from attribute types in the schema.

#### Schema
```
// Code here
```

#### Model
```
// Code here
```

#### General
```
// Code here
```

### GitHub linked issue:

Closes #1264 

### Type (select 1):
- [x] Bug fix
- [ ] Feature implementation
- [ ] Documentation improvement
- [ ] Testing improvement

- [ ] Test added to report bug (GitHub issue #--- @---)
- [ ] Something not listed here

### Is this a breaking change? (select 1):
- [ ] 🚨 YES 🚨
- [x] No
- [ ] I'm not sure

### Is this ready to be merged into Dynamoose? (select 1):
- [x] Yes
- [ ] No

### Other:
- [x] I have read through and followed the Contributing Guidelines
- [x] I have searched through the GitHub pull requests to ensure this PR has not already been submitted
- [x] I agree that all changes made in this pull request may be distributed and are made available in accordance with the [Dynamoose license](https://github.com/dynamoose/dynamoose/blob/master/LICENSE)
- [x] All of my commits and commit messages are detailed, explain what changes were made, and are easy to follow and understand
- [x] I have filled out all fields above

### CODE REVIEW

File: docs/docs/guide/Schema.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ const schema = new dynamoose.Schema({
...
 | Schema | False | M | True | True |   | This will be converted to an Object type. |
 | Model | Only if no `rangeKey` for model's schema | S \| N \| B \| M | True | If `rangeKey` in model's schema |   | Model Types are setup a bit differently. [Read below](#model-types) for more information. |
-| Combine | False | S | True | False | **attributes** - [string] - The attributes to store in the combine attribute.\n**seperator** - string (default: `,`) - The string used to seperate the attributes in the combine attribute. | When running `Model.update` you must update all the attributes in the combine attributes array, or none of them. This is to ensure your combine method remains in sync with your overall document. |
+| Combine | False | S | True | False | **attributes** - [string] - The attributes to store in the combine attribute. <br/>**seperator** - string (default: `,`) - The string used to seperate the attributes in the combine attribute. | When running `Model.update` you must update all the attributes in the combine attributes array, or none of them. This is to ensure your combine method remains in sync with your overall document. |
```
Comment (fishcharlie) [inline docs/docs/guide/Schema.md:93]:
```suggestion
| Combine | False | S | True | False | **attributes** - [string] - The attributes to store in the combine attribute.<br/>**seperator** - string (default: `,`) - The string used to seperate the attributes in the combine attribute. | When running `Model.update` you must update all the attributes in the combine attributes array, or none of them. This is to ensure your combine method remains in sync with your overall document. |
```

**Review by fishcharlie — COMMENTED** [review]
Let's see if this works.

**Review by fishcharlie — APPROVED** [review]
Looks great! Thanks for the contribution!

**Comment by MukulKolpe** [comment]
Thanks, @fishcharlie !!

**Pull request merged**